src/estimation/src/BipedFootContactClassifier.cpp

```diff
@@ -19,7 +19,7 @@ void BipedFootContactClassifier::updateFootContactState(double currentTime, doub
     m_leftFootContactClassifier.get()->contactMeasurementUpdate(currentTime, leftFootNormalForce);
     m_rightFootContactClassifier.get()->contactMeasurementUpdate(currentTime, rightFootNormalForce);
     m_leftFootContactState = m_leftFootContactClassifier.get()->contactState();
-    m_rightFootContactState = m_leftFootContactClassifier.get()->contactState();
+    m_rightFootContactState = m_rightFootContactClassifier.get()->contactState();
```

Comment:
Sorry, I should have spotted this during the review!

Reply:
My bad! Ill-effects of ctrl-c, ctrl-v. I hope there aren't more bugs :)